go.mod

```diff
@@ -171,6 +177,9 @@ require (
 replace (
 	// Use the keyring specified by the cosmos-sdk
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2 //indirect
+	github.com/cosmos/ibc-go/v7 => github.com/Stride-Labs/ibc-go/v7 v7.1.1-0.20230622125415-f45876a168a0
+	github.com/cosmos/interchain-security/v3 => github.com/Stride-Labs/interchain-security/v3 v3.1.1-0.20230713175550-227e13e60bac
```

Comment:
I had to add some necessary testing tools to ICS since there were blockers during running our unit tests.
I moved IBC simapp to ICS for setting up stride testing app correctly.
Older ICS versions had that simapp but for some reasons, it was removed recently so I moved it back for our unit tests.